core/src/createTextMaskInputElement.js

```diff
@@ -106,7 +107,8 @@ export default function createTextMaskInputElement(config) {
         pipe,
         placeholder,
         currentCaretPosition,
-        keepCharPositions
+        keepCharPositions,
+        showMask
```

Comment:
I don't think the `showMask` property is needed here for the `conformToMaskConfig` object.